tests/test_core.py

```diff
@@ -3767,6 +3767,11 @@ def test_dylink_memory_growth(self):
     self.set_setting('ALLOW_MEMORY_GROWTH', 1)
     self.do_basic_dylink_test()
 
+  @needs_dlfcn
+  def test_dylink_safe_heap(self):
+    self.set_setting('SAFE_HEAP', 1)
+    self.do_basic_dylink_test()
```

Comment:
Maybe we should run the complete SAFE_HEAP tests somewhere?

Reply:
They are quite slow, but yeah, might be worth doing at some low frequency.